src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> They can do things like wrap their contained items in their own renderer and so enable thing like accordions in lists.

How should this work? You have
List
..HeadingA
....Item1
....Item2
..HeadingB
....Item3
....Item4

If a single "group" is in the case e.g. HeadingA + Item1 + Item2, how could we achieve a behaviour where only one group is opened at a time? These kind of things are List behaviour in my opinion.

Another problem: It will be hard for the group render to decide on the level of heading it is currently in, if things are nested. h2, h3, h4?

If we use groups: How should the interface of the list look like? How are ungrouped items added? How are grouped items added? Are these two types of lists?